Caddyfile

```diff
@@ -5,6 +5,9 @@
     }
     cache {
         api {
+            prometheus {
+                enable true
+            }
```

Comment:
This is a bit awkward, an `enable` option is a bit redundant. If `prometheus` is there at all, you can assume they want it to be enabled.

I can't see any situation where you would want to configure it here but set it to `enable false`.

Using a boolean string is also awkward, it could just as easily just be `enable` without the boolean, to set it to true. But as I said above, I'm not convinced `enable` even needs to be there.

This example is also missing two of the other options that can be set for `prometheus`, so they should be added here too.

These comments also apply to the `souin` option below this one, which also uses `enable`.